seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -36,27 +36,14 @@
 class AnvilWorkspaceAPITest(AnvilAuthenticationTestCase):
     fixtures = ['users', 'social_auth', '1kg_project']
 
-    def _check_login_permissions(self, url, mock_logger):
-        # Test user doesn't login
-        response = self.client.post(url)
-        self.assertEqual(response.status_code, 302)
-        self.assertEqual(response.url, '/login/google-oauth2?next={}'.format(url))
+    def test_anvil_workspace_page(self, mock_logger):
+        # Requesting to load data from a workspace without an existing project
+        url = reverse(anvil_workspace_page, args=[TEST_WORKSPACE_NAMESPACE, TEST_NO_PROJECT_WORKSPACE_NAME])
+        collaborator_response = self.check_manager_login(url, login_redirect_url='/login/google-oauth2', policy_redirect_url='/accept_policies')
```

Comment:
Cool. I finally got how the `check_...login()` test utilities work.

Reply:
to be fair, we couldn't use the utility before because it checks that it fails for inactive users or users have not accepted the policies, and until this change those checks were failing